packages/teams-js/src/internal/utils.ts

```diff
@@ -338,3 +339,24 @@ export function createTeamsAppLink(params: pages.NavigateToAppParams): string {
   }
   return url.toString();
 }
+
+/**
+ * @hidden
+ * Checks if the Adaptive Card schema version to check for host support.
```

Comment:
How about: 
Checks if the Adaptive Card schema version is supported by the host